backend/specs/akvo/lumen/specs/import/values.clj

```diff
@@ -1,9 +1,17 @@
 (ns akvo.lumen.specs.import.values
   (:require [clojure.spec.alpha :as s]
             [akvo.lumen.util :refer (squuid)]
-            [akvo.lumen.specs :as lumen.s :refer (sample)]
+            [akvo.lumen.specs :as lumen.s]
             [akvo.lumen.lib.import.csv :as csv]))
 
+(defn round-coordinate
+  "Round coordinate to 6 digits but go by bigdec rounding to remove trailing
+   zeros. This to be able to compare facts to values from Postgis"
+  [coordinate]
+  (-> coordinate
+      bigdec
+      (.setScale 6 java.math.RoundingMode/HALF_EVEN)
```

Comment:
we can `(: import java.math.RoundingMode)` in the namespace declaration and the constant reads shorter? `RoundingMode/HALF_EVEN`